lib/devise.rb

```diff
@@ -521,8 +521,12 @@ def self.secure_compare(a, b)
     res == 0
   end
 
+  def self.deprecator
```

Comment:
We need to register the deprecator in the application deprecator. We want users to be able to silence the deprecations in their apps, or change the behavior to use notifications for example. Can you do that?

Reply:
I'll look into over the weekend when have some more free time.

Reply:
Done!